src/commcare_cloud/terraform/modules/openvpn/main.tf

```diff
@@ -5,7 +5,7 @@ resource aws_instance "server" {
   instance_type          = "${var.vpn_size}"
   subnet_id              = "${var.instance_subnet}"
   key_name               = "g2-access"
-  vpc_security_group_ids = ["${var.vpc-all-hosts-sg}","${var.g2-access-sg}","${aws_security_group.openvpn-access-sg.id}"]
```

Comment:
`openvpn-access-sg` is different from `ssh-sg`?

Reply:
This module is actually not yet used. I think G2 originally added it thinking they'd use it to set up our openvpn server, but that ended up getting set up in a way that's not captured here.

I'm still trying to figure out how we want to do VPNs for multiple environments so it's on my list to either use this or remove it from the code.